zk.js/tests/testData/tmp_test_psp -legacy.ts

```diff
@@ -0,0 +1,1142 @@
+export type TmpTestPsp = {
```

Comment:
can we rename the file to `tmp_test_psp_legacy.ts` or something similar without spaces? Typing this name in terminal is gonna be painful (`tmp_test_psp\ -legacy.ts` and on some terminals even that doesn't work)